src/App.vue

```diff
@@ -436,6 +436,9 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
 
         // parse draft filing into the store
         if (draftFiling) {
+          // const mocktype = 'ULC'
+          // draftFiling.business.legalType = mocktype
+          // draftFiling.incorporationApplication.nameRequest.legalType = mocktype
```

Comment:
For testing, I presume?
I don't mind if you commit this, but please add a comment we will easily find to remove this later.

Reply:
oh no no 😁 Let me take this out! Oopsies haha
this shouldn't be required for much longer hopefully!

Reply:
Removed!